app/javascript/packs/components/EditPopup.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import {
+  Modal,
+  Button,
+  FormGroup,
+  FormLabel,
+  FormControl,
+} from 'react-bootstrap';
+import Routes from 'Routes';
+
+import Fetch from './Fetch';
+
+export default class EditPopup extends React.Component {
+  state = {
+    task: {
+      id: null,
+      name: '',
+      description: '',
+      state: null,
+      author: {
+        id: null,
+        firstName: null,
+        lastName: null,
+        email: null,
+      },
+    },
+    isLoading: true,
+  };
+
+  loadCard = cardId => {
+    this.setState({ isLoading: true });
+    Fetch.get(Routes.apiV1TaskPath(cardId, { format: 'json' })).then(({ data }) => {
+      this.setState({ task: data });
+      this.setState({ isLoading: false });
+    });
+  };
+
+  componentDidUpdate(prevProps) {
+    const { cardId } = this.props;
+    if (cardId != null && cardId !== prevProps.cardId) {
+      this.loadCard(cardId);
```

Comment:
If you made this only for re-rendering edit form - you screwed up.
You should use keys https://reactjs.org/docs/lists-and-keys.html